simple-calculator/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>
+		My Calculator
+	</title>
+	<style type="text/css">
+		body
+		{
+			background-color:#F5DEB3;
+		}
+		form
+		{
+			
+			
+			margin-top: 100px;
+			margin-left: 1000px;
+			
+		}
+		.pi
+		{
+			height:60px;
+			text-align: center;
+			background-color: #FFF8DC;
+			border-style: none;
+			
+		}
+		.yi
+		{
+			border-radius: 25px;
+			background-color: #FFFFE0;
+			
+		}
+		div{
+			font-family:Courier New;
+			font-size: 30pt;
+
+		}
+	</style>
+</head>
+<script src="mycalculator.js"></script>
```

Comment:
It's weird to add something between `<head>` and `<body>`
you can move it inside body at the end.